tests/test_braid_vec.py

```diff
@@ -0,0 +1,684 @@
+from functools import partial
+import pytest
+import numpy as np
+
+# IMPORTANT: knpy should be installed first
+from knpy.braid_vec import Braid
+# from knpy.braid import BraidTransformation
```

Comment:
`BraidTransformation` is later used where it is not defined.

Reply:
Done